stdlib/2and3/ssl.pyi

```diff
@@ -49,23 +49,21 @@ def wrap_socket(sock: socket.socket, keyfile: Optional[str] = ...,
                 ciphers: Optional[str] = ...) -> SSLSocket: ...
 
 
-if sys.version_info < (3,) or sys.version_info >= (3, 4):
```

Comment:
The diff chunking algorithm sure makes a hash of this one. It's a lot clearer with whitespace-only changes excluded from the diff.